libraries/lwIP_Ethernet/src/LwipIntfDev.h

```diff
@@ -264,6 +269,24 @@ bool LwipIntfDev<RawDev>::config(const IPAddress& localIP, const IPAddress& gate
     }
     return true;
 }
+
+template<class RawDev>
+boolean LwipIntfDev<RawDev>::config(IPAddress local_ip, IPAddress dns) {
+
+    if (!local_ip.isSet()) {
+        return config(INADDR_ANY, INADDR_ANY, INADDR_ANY);
+    }
+    if (!local_ip.isV4()) {
+        return false;
+    }
+    IPAddress gw(local_ip);
+    gw[3] = 1;
```

Comment:
Notionally we support IPv6 as well as IPv4.  I'm not sure this assignment is valid in the IPv6 case.  Wouldn't we need to set the last octet of the 128 instead in that case?  (Asking as someone whose internet provider doesn't have IPv6 support so has never actually used it so apologies it it's a silly point!)

Reply:
there are no good defaults for IPv6 so the function has a comment with "IPv4 only" and
```
   if (!local_ip.isV4()) {
        return false;
```